code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -27,7 +27,7 @@
 	. = ..()
 	if(air_contents.gas[/datum/gas/oxygen] < 10)
 		. += "<span class='warning'>The meter on the [src] indicates you are almost out of oxygen!</span>"
-		playsound(usr, 'sound/effects/alert.ogg', 50, 1)
+//		playsound(usr, 'sound/effects/alert.ogg', 50, 1)
```

Comment:
if you wanna remove it, delete the line

Reply:
i figured i'd leave it in case someone wants to make it like, actually work how it's intended but sure